lib/sorcery/controller/config.rb

```diff
@@ -12,6 +12,8 @@ class << self
         attr_accessor :save_return_to_url
         # set domain option for cookies
         attr_accessor :cookie_domain
+        # Set token randomness
+        attr_accessor :token_randomness
```

Comment:
We can't access `Sorcery::Model::Config` instance directly because `Sorcery::Model::TemporaryToken` is a helper module and isn't included into `user_class`.

If you really want to do that, the code will be the following:

`Sorcery::Controller::Config.user_class.to_s.constantize.sorcery_config.token_randomness`

https://github.com/Sorcery/sorcery/blob/36698d7/lib/sorcery/adapters/active_record_adapter.rb#L22

Should I keep the setting in `Sorcery:Controler::Config` or do such trick?